docs/result.md

```diff
@@ -118,28 +146,26 @@ do_async(
 ) → Result[T, E]
 ```
 
-Async version of do. Example: 
+Async version of do. Example: ```
+``` python 
 
-``` final_result: Result[float, int] = await do_async(```
-     Ok(len(x) + int(y) + z)
-         for x in await get_async_result_1()
-         for y in await get_async_result_2()
-         for z in get_sync_result_3()
-     )
+final_result: Result[float, int] = await do_async(  Ok(len(x) + int(y) + z)  for x in await get_async_result_1()  for y in await get_async_result_2()  for z in get_sync_result_3()  ) 
+
```

Comment:
This rendered example doesn't look right. There's some issue with the docstring.

Tip, you can use this button to preview it in GitHub, in case you had not used it before
![image](https://github.com/rustedpy/result/assets/15352422/64649860-b70a-4bdc-ad3b-c55b1064f4cc)